### ISSUE TITLE
UI: Chat scrolls to top on certain unrelated actions

### ISSUE THREAD

**Issue by lemming552** [plainblack/Lacuna-Server-Open#377]
Seems to happen every? time I open up the forum from the left side bar the chat screen goes to the first entry.

**Comment by RedOrion** [comment]
Also happens occasionally when changing planets or hitting post for trades

**Comment by xHusky** [comment]
I agree with RedOrion: Happens to me when selecting a different colony.
